src/Networks/Blockcore.Networks.RoyalSportsCity/RoyalSportsCitySetup.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.Collections.Generic;
+using Blockcore.Consensus.Checkpoints;
+using Blockcore.Networks.RoyalSportsCity.Networks;
+using Blockcore.Networks.RoyalSportsCity.Networks.Setup;
+using NBitcoin;
+
+namespace Blockcore.Networks.RoyalSportsCity
+{
+    internal class RoyalSportsCitySetup
+    {
+        internal static RoyalSportsCitySetup Instance = new RoyalSportsCitySetup();
+
+        internal CoinSetup Setup = new CoinSetup
+        {
+            FileNamePrefix = "royalsportscity",
+            ConfigFileName = "royalsportscity.conf",
+            Magic = "01-4D-59-43",
+            CoinType = 6599,
+            PremineReward = 3000000000,
+            PoWBlockReward = 3000,
+            PoSBlockReward = 1000,
+            LastPowBlock = 25000,
+            MaxSupply = 21000000000,
+            GenesisText = "Decentralized Royal  Sports City",
+            TargetSpacing = TimeSpan.FromSeconds(64),
+            ProofOfStakeTimestampMask = 0x0000000F, // 0x0000003F // 64 sec
+            PoSVersion = 3
```

Comment:
If this is a new network, you should consider using PoS v4, perhaps you can give some instructions @dangershony ?

Reply:
I changed it to 4 .